devtools-ci-index.yaml

```diff
@@ -2034,6 +2034,15 @@
             ci_cmd: '/bin/bash ./cico/cico_run_EE_tests.sh ./cico/config_prod_preview'
             test_url: 'prod-preview.openshift.io'
             timeout: '20m'
+        - '{ci_project}-{git_repo}-build-che-credentials-{branch}':
+            git_organization: redhat-developer
+            git_repo: rh-che
+            ci_project: 'devtools'
+            branch: rh-che6
+            ci_cmd: '/bin/bash .ci/cico_build.sh && /bin/bash .ci/cico_deploy.sh'
+            saas_git: saas-openshiftio
+            saas_service_name: rh-che6
+            timeout: '30m'    
```

Comment:
why have you set the timeout to `30m` while others che build are `20m`? That's not critical but waiting 10 minutes more is really justified?

Reply:
@l0rd it looks like this timeout is not taken into account at all. However, it looks like the build of rh-che takes more than 20 mins in general. Last one took 23 mins for instance https://ci.centos.org/view/Devtools/job/devtools-rh-che-build-che-credentials-master/23/